mergify_engine/mergify_pull.py

```diff
@@ -130,6 +130,9 @@ def to_dict(self):
             #                    if s.state in ("pending", None)],
             "status-failure": [s.context for s in statuses
                                if s.state == "failure"],
+            "deletions": self.g_pull.deletions,
+            "additions": self.g_pull.additions,
+            "commits": self.g_pull.commits,
```

Comment:
I thought about that and I was wondering if we shouldn't expose a list of dict:

```
[
  {"author": "foobar", "files": […], "id": "abde123" },
  {"author": "foobar", "files": […], "id": "abde123" },
…]
```

so you could match on `commits.author` or `#commits`